components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="content-container">
   <div class="container">
-    <chef-loading-spinner *ngIf="orgsListLoading" size="50" fixed></chef-loading-spinner>
+    <div class="spinner">
+      <chef-loading-spinner *ngIf="orgsListLoading" size="50" fixed></chef-loading-spinner>
```

Comment:
@vivekshankar1 in this loader we are using the ```background-color: white``` and because of this, it was overriding the CSS of the save button. so I have added a parent class that will stop this overriding.

Reply:
Thanks, @himanshi-chhabra. That definitely fixed it. It's coming up properly now